app/views/views.html

```diff
@@ -1,9 +1,13 @@
 <div>
-  <div ng-class="$root.showViewList ? '' : 'hide-sidebar-list'">
+  <div ng-class="$root.showSideList ? '' : 'hide-sidebar-list'">
     <div class="page-container" ng-if="empty">
       <h1 class="cr-page-header">{{ 'VIEW.TITLE' | translate }}</h1>
       <div class="cr-panel-block">
-        There are no views created yet.
+        <p>
+          <b>There are no views created yet.</b>
+          <br /><br />
+          Use the <a href="https://crate.io/docs/crate/reference/en/latest/general/ddl/create-view.html">CREATE VIEW</a> statement to create new views.
```

Comment:
Change URL to `.../views.html` see https://github.com/crate/crate/pull/7138/files#diff-473eca00626203e4bf5df7059061162eR26